### PR TITLE
Extract getId / setId method from the MessageInterface to a new one

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️

### CODE REVIEW

File: src/Message/ParametrizedMessageInterface.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Message;
+
+interface ParametrizedMessageInterface extends MessageInterface
```
Comment (samdark) [inline src/Message/ParametrizedMessageInterface.php:7]: Name is a bit confusing. Parametrized means there are parameters. Here either there aren't or only one. Is there a better name?

Comment (xepozz) [reply src/Message/ParametrizedMessageInterface.php:7]: It was temporary named, thought about moving also `getMetadata`.

> Is there a better name?

Any suggestions?

Comment (samdark) [reply src/Message/ParametrizedMessageInterface.php:7]: `IdentifyableMessageInterface`?

Comment (xepozz) [reply src/Message/ParametrizedMessageInterface.php:7]: Or `MessageIdInterface`? :)

Also it might be closed after introducing envelope

**Pull request closed without merging**